examples/typescript-jest-node-fetch/src/github-api.test.ts

```diff
@@ -0,0 +1,32 @@
+/** @jest-environment setup-polly-jest/jest-environment-node */
+import autoSetupPolly from "./utils/auto-setup-polly";
+import { getUser } from "./github-api";
+
+// 🟡 Important: Run `autoSetupPolly()`/`setupPolly()` EARLY!
+let pollyContext = autoSetupPolly();
```

Comment:
Should we follow the docs for `setup-polly-jest` and add this within a `describe` block?